app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
+      log_in user
+      if params[:session][:remeber_me] == "1"
+        remember(user)
```

Comment:
```suggestion
        remember user
```
chỗ nào bỏ dc ngoặc mà vẫn chạy thì bỏ nhé e